formatTest/unit_tests/expected_output/jsx.re

```diff
@@ -270,3 +270,5 @@ let x = foo /></ bar;
 <div onClick=this##handleClick>
   <> foo bar </>
 </div>;
+
+<Foo> 1 2 other </Foo>;
```

Comment:
Small question, am I wrong that I would expect `<Foo> [1, 2] other </Foo>` here?

Reply:
yeah I'm not sure about this one. it currently looks to me that lists are always spliced in

Reply:
We should not eagerly cancel out list spread + list wrapping at a syntax level, since there might be ppxes that turns the list literal into something else (e.g. react jsx ppx turns children list into array. So `<Foo> ...[] </Foo>` should stay as such after formatting. Though I recall the implementation for keeping it like that becomes complicated?

Reply:
that isn't the case in master however. this is the current behavior:

```
$ echo '<div> ...[] </div>' | refmt
<div />;
```

Reply:
Looks like @jordwalke's comment (https://github.com/facebook/reason/pull/1429#issuecomment-334621175) is exactly the behavior that this PR maintains?